dev/kube/klog.sh

```diff
@@ -94,57 +102,59 @@ function handle_container() {
     fi
 
     retrieve_container_kube_logs 2> /dev/null || true
-    printf "\n"
+    printf '\n'
 }
 
 function get_pod_description() {
-  printf "  Descriptions ...\n"
+  printf '  Descriptions ...\n'
   kubectl describe pods "${POD}" --namespace "${NS}" > "${POD_DIR}/describe-pod.txt"
 }
 
 function get_all_resources() {
-  printf "Resources ...\n"
+  printf '  Resources ...\n'
   kubectl get all --export=true --namespace "${NS}" --output=yaml > "${KLOG}/${NS}/resources.yaml"
 }
 
 function get_all_events() {
-  printf "Events ...\n"
+  printf '  Events ...\n'
   kubectl get events --export=true --namespace "${NS}" --output=yaml > "${KLOG}/${NS}/events.yaml"
 }
 
 rm -rf "${KLOG:?}/${NS:?}"
 NAMESPACE_DIR="${KLOG}/${NS}"
 
 # Iterate over pods and their containers.
-PODS=($(get_all_the_pods))
+PODS="$(get_all_the_pods)"
 
-for POD in "${PODS[@]}"; do
+for POD in ${PODS}; do
```

Comment:
Since we don't re-use it, `for POD in $(get_all_the_pods); do` should work too.

Reply:
Fixed with commit b6e4b4ddb570770345aa1acdb867ed2c093ed925